shared/css/conversation.css

```diff
@@ -107,38 +107,11 @@ html[dir="rtl"] .conversation-toolbar-media-btn-group-box > button:first-child:h
   content: url("../img/audio.svg");
 }
 
-.btn-mute-audio.muted:after {
-  content: url("../img/audio-mute.svg");
```

Comment:
I believe all of the svgs listed below can be removed as part of removing the css:

- `audio-mute.svg`
- `audio-hover.svg`
- `audio-mute-hover.svg`
- `video-hover.svg`
- `video-mute.svg`
- `video-mute-hover.svg`